packages/core/src/domain/ai-brief/modify-resources.ts

```diff
@@ -243,12 +243,24 @@ export type SlimPractitioner = Omit<Practitioner, "name" | "qualification" | "ad
 function getSlimPractitioner(res: Practitioner): SlimPractitioner {
   const updRes = cloneDeep(res);
   const name = getNameString(updRes.name);
+  const names: string[] = [];
+
+  if (name && name.length > 0) {
+    const nameParts = name.split(" ");
+    // Taking only the first few practitioners, in case the whole care team is present in the list
+    if (nameParts.length > 10) {
+      names.push(nameParts.slice(0, 10).join(" "));
+    } else {
+      names.push(name);
+    }
+  }
```

Comment:
Why don't we instead limit the number of items from `updRes.name` that we send to `getNameString()`?

With this approach it seems that we're mixing people's names, no?

Example:
- original `.name`
  - `John Doe`
  - `Martin Ducatti`
- `name`
   - `John Doe Martin Ducatti`
- `names`
   - `John Doe Martin`